chart/values.yaml

```diff
@@ -319,6 +319,16 @@ extraEnvFrom: ~
 # Airflow database & redis config
 data:
   # If secret names are provided, use those secrets
+  # These secrets must be created manually, eg:
+  #
+  # kind: Secret
+  # apiVersion: v1
+  # metadata:
+  #   name: custom-airflow-metadata-secret
+  # type: Opaque
+  # data:
+  #   connection: base64_encoded_connection_string
+
   metadataSecretName: ~
   resultBackendSecretName: ~
```

Comment:
Might be worth a comment that this will use `metadataSecretName`, if it's set.

Reply:
Currently it won't because you have to create two separate secrets that are almost the same except that one has db+ prefix - that's the reason for this pr :D Added info that for airflow 2.4.0 and above it will